source/win-main/file-manager/TreeItem.vue

```diff
@@ -416,7 +416,14 @@ watch(toRef(props, 'obj'), function (value) {
   updateObject(value)
 })
 
-onMounted(uncollapseIfApplicable)
+onMounted(() => {
+  ipcRenderer.on('shortcut', (_, message) => {
+    if (message === 'new-dir') {
+      operationType.value = 'createDir'
+    }
+    uncollapseIfApplicable()
+  })
+})
```

Comment:
Almost – this function needs to be called OUTSIDE of the `ipcRenderer.on`-callback. Then it's good to go. See my suggestion.

```suggestion
onMounted(() => {
  uncollapseIfApplicable()
  ipcRenderer.on('shortcut', (_, message) => {
    if (message === 'new-dir') {
      operationType.value = 'createDir'
    }
  })
})
```